### PR TITLE
Document how we run meetings

### PR DESCRIPTION
**Pull request by ewilderj**
I created a directory that can hold our procedural how-tos. If we document things we do ASAP then it makes it easy for others to help out and contribute. To start off, I've documented how to run community meetings. Other things we could put in here might be "how to give a talk on Kubeflow", etc, etc.

### CODE REVIEW

File: how-to/community_meetings.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+International numbers available: https://zoom.us/zoomconference?m=Os1EjlUlpb2_XUMaQ6dX1azqMK5CkfWH
+```
+
+
+
+
+## Running the meeting
+
+Start the Zoom meeting 5 minutes before its assigned time.  The meeting should
+be configured that new arrivals are muted by default, to save noise.  Usually,
```
Comment (jlewi) [inline how-to/community_meetings.md]: nit: "save noise" -> "reduce noise"

File: how-to/community_meetings.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+* Leave the doc editable for a few hours for incoming amendments, then change
+  the sharing so the group has *Comment & Suggest* permissions. This is to avoid
+  us having worry about after-the-fact changes that the community doesn't get
+  notified of.
+  
+## That's it!
+
+If you have suggestions or improvements, please submit a PR to this
+documentation or discuss it on the group mailing list.
+
```
Comment (jlewi) [inline how-to/community_meetings.md]: nit delete some of the blank lines.

Comment (ewilderj) [reply how-to/community_meetings.md]: Fixed. Fell asleep on the Enter key...

File: how-to/community_meetings.md (hunk 1 above)
Comment (ewilderj) [reply how-to/community_meetings.md]: Fixed. Maybe my British idiom…

**Pull request merged**